csharp/ql/test/library-tests/dataflow/collections/CollectionFlow.cs

```diff
@@ -481,4 +481,26 @@ public void CollectionExpressionSpreadElementNoFieldFlow()
         IntegerCollection ic1 = [.. ic0];
         Sink(ic1.Payload); // No flow
     }
+
+    public void SpanConstructorFlow()
+    {
+        var a = new A();
+        Span<A> span = new Span<A>(ref a);
+        Sink(span[0]); // flow
+    }
+
+    public void SpanToArrayFlow()
+    {
+        var a = new A();
+        Span<A> span = new Span<A>(ref a);
+        var arr = span.ToArray();
+        Sink(arr[0]); // flow
+    }
+
+    public void SpanFillFlow(Span<A> target)
+    {
+        var a = new A();
+        target.Fill(a);
+        Sink(target[0]); // flow
```

Comment:
@hvitved : Flow is not identified correctly in this case. The test query looks fine, the summary models appear to correctly apply to the methods and a model exists for `Fill`.
However `Span<T>` is a struct - do you know if that somehow could be the reason?

Reply:
It looks like this is related to missing post update notes.

Reply:
Yes, you are right we need a post-update node.

Reply:
This was improved here: https://github.com/github/codeql/pull/15502